articles/techniques/operations-and-functions.md

```diff
@@ -263,7 +263,7 @@ This new function is indeed a function, in that if we call it with the same valu
 Thus, the decoder can be safely run inside operations without having to reason about how the decoding logic interacts with the definitions of the different operation specializations.
 That is, we have isolated the classical logic inside a function, guaranteeing to the compiler that the function call can be reordered with impunity so long as the input is preserved.
 
-We can also treat functions as first-class values, as we will see in more detail when we discuss [operations and function types](#operation-and-function-types).
+We can also treat functions as first-class values, as we will see in more detail when we discuss [operations and function types](#operations-and-function-types).
```

Comment:
I don't see a heading with this anchor in `operations-and-functions.md`, such that this link is still broken?

Reply:
This reference was broken in articles\relnotes\index.md:  Add a way to specify a short name for namespaces in [open directives](xref:microsoft.quantum.language.file-structure#open-directives).
Need to check after review build.

Reply:
That's why I removed this sentence.  The heading doesn't exist.  I may be missing something here.